wellregistry/registry/tests/lookups.py

```diff
@@ -0,0 +1,56 @@
+"""
+Module for Managing lookups used in tests.
+"""
+
+from ..models import AgencyLovLookup, AltDatumDimLookup, CountryLookup, CountyLookup, HorzDatumDimLookup
+from ..models import NatAqfrLookup, StateLookup, UnitsDimLookup
+
+class LookupData:
```

Comment:
Curious as to why you created this class. All of the methods are static. You do not need to create classes in Python so these methods could just be declared as methods in the python module.

Reply:
looks like it is used for the test case below.

Reply:
As a said above, I'm questioning whether it needs to be a class or whether the methods can just be declared in the module. Methods in python do not need to be part of a class.